src/test/unittest/utils.py

```diff
@@ -20,6 +20,10 @@
 
 def require_architectures(*archs):
     """Enable test only for specified architectures"""
+    if sys.platform == 'win32':
```

Comment:
small nit, feel free to disregard:
I'd prefer to "normalize" the output of `platform.machine()` instead of rewriting the `archs` array, which kinda feels backward to me.